src/TheThingsNetwork.h

```diff
@@ -6,13 +6,35 @@
 
 #include <Arduino.h>
 #include <Stream.h>
+#include <avr/pgmspace.h>
 
 #define TTN_DEFAULT_SF 7
 #define TTN_DEFAULT_FSB 2
-#define TTN_RETX 7
+#define TTN_RETX "7"
 
-#define TTN_PWRIDX_868 1
-#define TTN_PWRIDX_915 5
+#define TTN_PWRIDX_868 "1"
+#define TTN_PWRIDX_915 "5"
+
+#define TTN_BUFFER_SIZE 300
```

Comment:
Please check if there are more defines that can be moved to `.cpp` to not be public. Only those needed to change default settings should be here, and also added to `keywords.txt`.